### PR TITLE
Client: Added left-handed mode and viewmodel FOV & xyz-offset cvars

### PR DESCRIPTION
**Pull request by SmileyAG**
Based on:

https://github.com/YaLTeR/OpenAG/commit/8df40eaf03a01f2b2c218e45fe0a539d134b781a
https://github.com/YaLTeR/OpenAG/pull/129
https://github.com/YaLTeR/OpenAG/pull/107
https://github.com/YaLTeR/OpenAG/pull/108
https://github.com/YaLTeR/OpenAG/pull/109
https://github.com/YaLTeR/OpenAG/pull/111
https://github.com/YaLTeR/OpenAG/commit/3b60ea73db813103f55c2f9d509c7fa7ff6719db

### CODE REVIEW

File: src/game/client/ev_common.cpp [hunk 1]
```diff
@@ -175,7 +175,12 @@ void EV_GetDefaultShellInfo(event_args_t *args, float *origin, float *velocity,
 		}
 	}
 
-	fR = gEngfuncs.pfnRandomFloat(50, 70);
+	extern ConVar cl_righthand;
+	if (cl_righthand.GetFloat() > 0)
+		fR = gEngfuncs.pfnRandomFloat(-50, 70);
```
Comment (tmp64) [inline src/game/client/ev_common.cpp:180]: Shouldn't this be
```cpp
fR = -gEngfuncs.pfnRandomFloat(50, 70);
```
?

File: src/game/client/ev_hldm.cpp [hunk 2]
```diff
@@ -502,7 +504,10 @@ void EV_FireGlock1(event_args_t *args)
 		V_PunchAxis(0, -2.0);
 	}
 
-	EV_GetDefaultShellInfo(args, origin, velocity, ShellVelocity, ShellOrigin, forward, right, up, 20, -12, 4);
+	if (cl_righthand.GetFloat() > 0)
```
Comment (tmp64) [inline src/game/client/ev_hldm.cpp:507]: I think it's better to negate Z velocity in `EV_GetDefaultShellInfo` instead of every place that it is called at

**Review by tmp64 — CHANGES_REQUESTED** [review]

**Comment by tmp64** [comment]
Can you please split this into two PRs? I'll be pushing a commit later today that adds support for OpenGL 2.0 which doesn't require linking with opengl32.dll or GLX. This will cause a mess during rebasing and updating ea9fb7152d1547e6b33dcdc2bc943780b93268dc.

File: src/game/client/ev_common.cpp (hunk 1 above)
Comment (SmileyAG) [reply src/game/client/ev_common.cpp:180]: Oops. I'll fix it in next PR.

File: src/game/client/ev_hldm.cpp (hunk 2 above)
Comment (SmileyAG) [reply src/game/client/ev_hldm.cpp:507]: Yeah it could be formatted better

**Comment by SmileyAG** [comment]
> 
> 
> Can you please split this into two PRs? I'll be pushing a commit later today that adds support for OpenGL 2.0 which doesn't require linking with opengl32.dll or GLX. This will cause a mess during rebasing and updating [ea9fb71](https://github.com/tmp64/BugfixedHL-Rebased/commit/ea9fb7152d1547e6b33dcdc2bc943780b93268dc).

Alright.

**Pull request closed without merging**